assets/css/_diversion_page.scss

```diff
@@ -54,3 +54,32 @@
     }
   }
 }
+
+.c-diversion-panel__origin,
+.c-diversion-panel__direction {
+  font-size: 14px;
+}
+
+.c-diversion-panel__description {
+  font-weight: 600;
+}
+
+.l-diversion-page__header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.c-diversion-panel__h2 {
+  font-size: 24px;
+  font-weight: 600;
+}
+
+.c-diversion-panel__h3 {
+  font-size: 20px;
+  font-weight: 600;
+}
+
+.c-diversion-panel__help-text {
+  line-height: 24px;
```

Comment:
question: can unitless values be used instead here?
https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values

Reply:
Thanks for catching that! Did that and a few other CSS things here - https://github.com/mbta/skate/pull/2377/commits/b4f29bbd9c77cf9b409366e0cf73870128eeaff9